### PR TITLE
Adding domain alert missing parity from web app

### PR DESCRIPTION
**Pull request by rmhowe425**
- Closes out #140 
- Added checks to filter out private IPv4 and IPv6 addresses when creating a domain-based alert.
- Added checks to control output messages to the user when creating  a domain-based alert.

### CODE REVIEW

**Comment by rmhowe425** [comment]
@achillean PR is ready for inspection!

File: shodan/cli/alert.py [hunk 1]
```diff
@@ -132,22 +134,30 @@ def alert_domain(domain, triggers):
...
-            domain_ips = set([record['value'] for record in info['data']])
+            domain_ips = set([record['value'] for record in info['data']
+                              if not ip_address(record['value']).is_private])
 
-        # Create the actual alert
-        click.secho('Creating alert...', dim=True)
-        alert = api.create_alert('__domain: {}'.format(domain), list(domain_ips))
+        if not domain_ips:
+            flag = False
+            click.secho('No external IPs were found to be associated with this domain. '
```
Comment (achillean) [inline shodan/cli/alert.py:145]: Why not just `raise click.ClickException('No external ...')`? Then we don't need to use a `flag` variable to track the return status and I think it's technically correct to error out if it wasn't possible to generate an asset group due to lacking IPs.

Comment (rmhowe425) [reply shodan/cli/alert.py:145]: ahhh yep! I'll make that change!

Comment (rmhowe425) [reply shodan/cli/alert.py]: @achillean Makes sense! Changes made!

**Pull request merged**